lib/pymedphys/_experimental/streamlit/apps/sum_doses.py

```diff
@@ -0,0 +1,241 @@
+"""Copyright (C) 2021 Matthew Jennings
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import copy
+import os
+import pathlib
+from typing import Sequence, Union
+
+from pymedphys._imports import streamlit as st
+
+import numpy as np
+
+import pydicom
+from pydicom.errors import InvalidDicomError
```

Comment:
```suggestion
```

Then, add `import pydicom.errors` over here:

https://github.com/pymedphys/pymedphys/blob/c5fa262fd5eb927dcbbe1fced3469f13ad7f1a96/lib/pymedphys/_imports/imports.py#L75

And when using `InvalidDicomError` instead call it as `pydicom.errors.InvalidDicomError`

Reply:
Thanks!